src/main/resources/db/migration/V202210040942__create_table_project_history.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE project_history (
+    project_history_id NUMBER(6) PRIMARY KEY,
+    project_id NUMBER(6) NOT NULL,
+    employee_id NUMBER(6) NOT NULL,
```

Comment:
make these columns to be PRIMARY KEY (add a constraint), because the combination of project_id and employee_id will always be unique, and then delete project_history_id